.github/workflows/iroha2-release.yml

```diff
@@ -2,7 +2,7 @@ name: I2::Release::Publish
 
 on:
   push:
-    branches: [iroha2-stable, iroha2-lts]
+    branches: [stable, lts]
```

Comment:
`iroha2-lts` wasn't renamed to `lts` as i can see

Reply:
tbh `lts` stuff quite is irrelevant now

Reply:
It's just a plug for a future possible `lts` branch that could be appeared. I can remove it.

Reply:
So, should it be pinned to the old `iroha2-lts` or to be removed completely?

Reply:
pin it to `iroha2-lts`, but we're going to remove it later

Reply:
Okay, but step `name: Get tag from branch name` will now work for `iroha2-lts` branch

Reply:
it's not relevant, we're removing `lts` branch anyhow

Reply:
Done